tools/cmaketobzl/cmaketobzl.go

```diff
@@ -308,6 +371,11 @@ func (e *eval) CurrentDirectory() string {
 	return path.Join(e.path...)
 }
 
+// isTopLevel returns true in a top-level CMakeLists.txt file.
+func (e *eval) isTopLevel() bool {
+	return e.v.Depth() == 0 || path.Join(e.ProjectRoot(), e.CurrentDirectory()) == e.ProjectRoot()
```

Comment:
Is this meant to be `&&`? The comment says "top-level" but I'm not sure whether that also includes nested projects. The second clause seems to imply that.

Reply:
No; the issue here is that cmaketobzl has slightly different semantics than cmake itself, because it needs to support being provided with explicit subdirectories.  Also, in the currently implementation Depth() will never be 0 because we always "enter" the root directory, which increases the depth.

Reply:
This is necessitated by the incomplete implementation of the CMake semantics (particularly `foreach`) so in order to import the LLVM/Clang build system we need to specifically include both the root and target-specific subdirectories.